src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import { getUser } from '../../api';
+
+import { Todo } from '../../types/Todo';
+import { TodoModal } from '../TodoModal';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const [modalIsOpen, setModalIsOpen] = useState(false);
+  const showModal = () => setModalIsOpen(!modalIsOpen);
+  const {
+    title,
+    id,
+    completed,
+    userId,
+  } = todo;
+
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    getUser(userId)
```

Comment:
The `getUser` API call should have error handling. Consider adding a `.catch` block to handle potential errors.